README.md

```diff
@@ -2,6 +2,9 @@
 
 **Spotify has disabled my API access for the skill, it was violating their Terms of Service by enabling voice control. I must have missed this back in 2017 when I created the skill.**
 
+To use this skill currently you will need to create your own "application using
+https://developer.spotify.com". See instructions under the heading "Personal access token"
```

Comment:
this wasn't a happy find for me;

I couldn't find anything from there 

tried searching https://developer.spotify.com/?query=personal%20access%20token
then found: https://developer.spotify.com/documentation/web-playback-sdk/quick-start/

Reply:
I don't think even that is the right spot; we might be missing terminology.  Later the script says:
> "requires the user to go to developer.spotify.com and set up a
      developer account, create an "Application" and make sure to whitelist
      "https://localhost:8888""
, where do we create an application?

Reply:
k found: https://developer.spotify.com/documentation/general/guides/app-settings/

Reply:
nice set of changes to the readme thx! - LGTM ; i think it addresses the "user gap" (at the spotify dev app side)